marlowe-runtime-cli/changelog.d/20230620_075705_brian.bush_PLT_5817.rst

```diff
@@ -0,0 +1,4 @@
+Added
+-----
+
+- Create command outputs results of contract safety analysis.
```

Comment:
This shouldn't have its own `changelog.d` folder - this is part of the `marlowe-runtime` component, not a separate one

Reply:
Thanks. We should probably document which folders should not have change logs.

Reply:
Fixed in 34f1d7e5d89e68996254f165386a3264372c3252.